src/Compilers/CSharp/Test/Syntax/Diagnostics/DiagnosticTest.cs

```diff
@@ -331,6 +330,9 @@ public void WarningLevel_2()
                         case ErrorCode.WRN_InvalidVersionFormat:
                             Assert.Equal(4, ErrorFacts.GetWarningLevel(errorCode));
                             break;
+                        case ErrorCode.WRN_NubExprIsConstBool2:
```

Comment:
nit: May be worth adding a comment in test to indicates cases below, with warning level 5+, are "warning waves"